src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IConditionalAccessExpression.vb

```diff
@@ -0,0 +1,65 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.Test.Utilities
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.Semantics
+
+    Partial Public Class IOperationTests
+        Inherits SemanticModelTestBase
+
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact()>
+        Public Sub IConditionalAccessExpression_SimpleMethodAccess()
+            Dim source = <![CDATA[
+Option Strict On
+
+Public Class C1
+    Public Sub M1()
+        Dim o As New Object
+        o?.ToString()'BIND:"o?.ToString()"
+    End Sub
+End Class]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IConditionalAccessExpression (OperationKind.ConditionalAccessExpression, Type: System.Void) (Syntax: 'o?.ToString()')
+  Expression: ILocalReferenceExpression: o (OperationKind.LocalReferenceExpression, Type: System.Object) (Syntax: 'o')
+  WhenNotNull: IInvocationExpression (virtual Function System.Object.ToString() As System.String) (OperationKind.InvocationExpression, Type: System.String) (Syntax: '.ToString()')
+      Instance Receiver: IPlaceholderExpression (OperationKind.PlaceholderExpression, Type: System.Object) (Syntax: 'o?.ToString()')
```

Comment:
>IPlaceholderExpression [](start = 25, length = 22)

Probably should be IConditionalAccessInstanceExpression. Do we have a separate issue tracking that?